CODEOWNERS

```
@@ -1,4 +1,4 @@
 /k-distribution/include/kframework/builtin/* @dwightguth @ehildenb
 /Jenkinsfile @runtimeverification/admin
 /.github/workflows/* @runtimeverification/admin
-/package/version @dwightguth @ehildenb @radumereuta
+/package/version @dwightguth @ehildenb @radumereuta @runtimeverification/jenkins @F-WRunTime @Baltoli
```

This is the correct RV Jenkins user, right? https://github.com/rv-jenkins

```suggestion
/package/version @dwightguth @ehildenb @radumereuta @rv-jenkins @F-WRunTime @Baltoli
```